srcpkgs/task/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'task'
 pkgname=task
-version=2.6.2
+version=3.0.2
 revision=1
 build_style=cmake
-makedepends="libuuid-devel gnutls-devel"
+makedepends="libuuid-devel gnutls-devel rust cargo"
```

Comment:
you likely want to add them to *host*makedepends

Reply:
Thanks for the suggestion. Looks like it got us a little bit further. Not sure what to do about the missing target now...

Reply:
you should look into rust build_helper

Reply:
Thanks for the help! This got me a bit further indeed. Now it looks like there is a bug in taskwarrior in 32bits platforms. I'm tempted to dig into taskwarrior's code to see if I can fix it and PR it there, either that or I will at least file a bug report. Not sure what's your policy about what we need to do for this PR? Is it possible to release the package for part of the arch only?

Reply:
add a patch that does this: https://github.com/void-linux/void-packages/actions/runs/9259003744/job/25470099738?pr=50527#step:7:549